frontend/src/main/components/Rider/RiderTable.js

```diff
@@ -0,0 +1,175 @@
+import React from "react";
+import OurTable, { ButtonColumn } from "main/components/OurTable";
+import { useBackendMutation } from "main/utils/useBackend";
+import { cellToAxiosParamsDelete, onDeleteSuccess } from "main/utils/riderUtils"
+import { useNavigate } from "react-router-dom";
+import { hasRole } from "main/utils/currentUser";
+
+export default function RiderTable({
```

Comment:
Is this a RiderTable or a RidesTable?

Reply:
[007145b](https://github.com/ucsb-cs156-s23/proj-gauchoride-s23-5pm-4/pull/32/commits/007145b7af202a2285f77a7ca76d7e8f4b67990f) should fix this